compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -1273,7 +1280,6 @@ void t_java_generator::generate_tuple_scheme_read_value(ostream& out, t_struct*
   indent_up();
   indent(out) << "switch (setField) {" << endl;
   indent_up();
-
```

Comment:
I think you lost a line in your change. I'm not sure if clang-format would re-add it. It'd be good to re-run it after your changes, just to be sure this PR is still well-formatted.